coops-ui/src/components/AnnualReport/RegisteredOfficeAddress.vue

```diff
@@ -166,6 +167,23 @@ export default class RegisteredOfficeAddress extends Vue {
     return this.formValid
   }
 
+  /**
+   * Emits updated addresses object to the parent page.
+   */
+  @Emit('update:addresses')
+  private emitAddresses (): object {
+    return { deliveryAddress: this.deliveryAddress, mailingAddress: this.mailingAddress }
+  }
+
+  /**
+   * Called when addresses property changes (ie, when parent page has loaded a draft filing).
+   */
+  @Watch('addresses')
```

Comment:
Instead of watching addresses and updating deliveryAddress and mailingAddress, can't we just use the sub-objects within addresses?

Reply:
Yes we can read the sub-objects but we can't write to them (Vue error). See:
https://stackoverflow.com/questions/39868963/vue-2-mutating-props-vue-warn
https://michaelnthiessen.com/avoid-mutating-prop-directly/